### DISCUSSION TITLE
Error: EPERM: operation not permitted, mkdir

### DISCUSSION THREAD

**Discussion by royalkd** [aclap-dev/video-downloadhelper#97] [Help and Q&A]
### Your question

as title
the error apears when I download any films

Error: EPERM: operation not permitted, mkdir 'D:\'
    at rs.receive (chrome-extension://jmkaglaafmhbcpleggkmaliipiilhldn/background/main.js:1:13157)
    at chrome-extension://jmkaglaafmhbcpleggkmaliipiilhldn/background/main.js:1:42563

I changed my folder /security tab to full control, its not worked.

**Comment by paulrouget** [comment]
Firefox or Chrome?

**Reply by royalkd** [reply]
edge

**Comment by paulrouget** [comment]
Closing this in favor of https://github.com/aclap-dev/video-downloadhelper/discussions/91

**Reply by royalkd** [reply]
got it
